runtime-tests/fastly/vitest.config.ts

```diff
@@ -1,17 +1,10 @@
-/// <reference types="vitest" />
 import fastlyCompute from 'vite-plugin-fastly-js-compute'
-import { defineConfig } from 'vitest/config'
-import config from '../../vitest.config'
+import { defineProject } from 'vitest/config'
 
-export default defineConfig({
+export default defineProject({
   plugins: [fastlyCompute()],
   test: {
     globals: true,
-    include: ['**/runtime-tests/fastly/**/(*.)+(test).+(ts|tsx)'],
-    exclude: ['**/runtime-tests/fastly/vitest.config.ts'],
-    coverage: {
-      ...config.test?.coverage,
-      reportsDirectory: './coverage/raw/runtime-fastly',
-    },
```

Comment:
Project configuration is simpler as test files are found relative to the `vitest.config.ts`, while `coverage` configuration remains in the root `vitest.config.ts`